packages/core/src/Lib/UseFlyonUIVueAppConfig/Types/FlyonUIVueAppConfig.ts

```
@@ -1,13 +1,13 @@
-import type { TableProps }                                                       from '@/UI/Tables';
-import type { Ref }                                                              from 'vue';
+import type { Color, Colorable, ConfigurableComponentName, Preset, Shape, Size }       from '@/Lib';
+import type { PickIfExists, Prettify }                                                 from '@/Lib/Utils/Internal';
+import type { Direction, HorizontalPosition }                                          from '@/Types';
+import type { BadgeProps, ButtonProps, LabelType, LoadingProps, MenuProps }            from '@/UI/Components';
+import type { KeyboardProps, LinkProps }                                               from '@/UI/Content';
+import type { IconProps }                                                              from '@/UI/Customization';
+import type { CheckboxProps, InputTextProps, SelectProps, SwitchProps, TextareaProps } from '@/UI/Forms';
+import type { ModalProps, TooltipProps }                                               from '@/UI/Overlays';
+import type { TableProps }                                                             from '@/UI/Tables';
+import type { Ref }                                                                    from 'vue';
```

ESLint

[error] Expected "vue" (type) to come before "@/UI/Tables" (internal-type).

(perfectionist/sort-imports)